Source/Tools/Flax.Build/Projects/VisualStudio/VCProjectGenerator.cs

```diff
@@ -175,9 +176,9 @@ public override void GenerateProject(Project project)
                 vcProjectFileContent.AppendLine("    <LibraryWPath />");
                 vcProjectFileContent.AppendLine("    <SourcePath />");
                 vcProjectFileContent.AppendLine("    <ExcludePath />");
-                vcProjectFileContent.AppendLine(string.Format("    <NMakeBuildCommandLine>{0} -build</NMakeBuildCommandLine>", cmdLine));
-                vcProjectFileContent.AppendLine(string.Format("    <NMakeReBuildCommandLine>{0} -rebuild</NMakeReBuildCommandLine>", cmdLine));
-                vcProjectFileContent.AppendLine(string.Format("    <NMakeCleanCommandLine>{0} -clean</NMakeCleanCommandLine>", cmdLine));
+                vcProjectFileContent.AppendLine(string.Format("    <NMakeBuildCommandLine>dotnet {0} -build</NMakeBuildCommandLine>", cmdLine));
```

Comment:
Why not run the build tool directly here?

Reply:
I think this was leftover code from the early versions where the build tool got built as a DLL-only, and you'd need to use `dotnet` to call DLL-binaries.